app/controllers/products_controller.rb

```diff
@@ -25,20 +27,19 @@ def create
   end
 
   def sellerProduct
-      @sellerProduct = current_user.seller.products.all
+    @sellerProduct = current_user.seller.products.all
```

Comment:
You are passing all products so variable name should be @seller_products.

Reply:
thank you for notify me i will update it